In [2]:
numerical_data ={
    "Product_price": 250.00,
    "Order_quantity":2,
    "shipping_fees": 28.00,
    "total_invoice": 528.00
}
categorical_data = {
    "payment_method": "تمارا",
    "order_status": "تم الشحن",
    "customer_city": "الرياض",
    "product_category": "عطور"
}
text_data ={
    "customer_name": "أحمد محمد العتيبي",
    "delivery_address": "حي النرجس، شارع عثمان بن عفان، مبنى 14",
    "customer_note": "يرجى الاتصال قبل التوصيل، الطلب عبارة عن هدية",
    "product_review": "منتج ممتاز، جودة عالية وسعر مناسب، سأوصي به لأصدقائي"
    }
print("📊 [1] Numerical Data:")
for key, value in numerical_data.items():
      if key == "order_quantity":
        print(f"   - {key}: {value} قطع")
      else:
        print(f"   - {key}: {value} ريال سعودي")

print("\n📌 [2] Categorical Data:")
for key, value in categorical_data.items():
    print(f"   - {key}: {value}")

print("\n📝 [3] Text Data:")
for key, value in text_data.items():
    print(f"   - {key}: {value}")

📊 [1] Numerical Data:
   - Product_price: 250.0 ريال سعودي
   - Order_quantity: 2 ريال سعودي
   - shipping_fees: 28.0 ريال سعودي
   - total_invoice: 528.0 ريال سعودي

📌 [2] Categorical Data:
   - payment_method: تمارا
   - order_status: تم الشحن
   - customer_city: الرياض
   - product_category: عطور

📝 [3] Text Data:
   - customer_name: أحمد محمد العتيبي
   - delivery_address: حي النرجس، شارع عثمان بن عفان، مبنى 14
   - customer_note: يرجى الاتصال قبل التوصيل، الطلب عبارة عن هدية
   - product_review: منتج ممتاز، جودة عالية وسعر مناسب، سأوصي به لأصدقائي


<div dir="rtl">

# <center>📊 تقرير تصنيف بيانات متجر سلة الإلكتروني — السوق السعودي</center>

---

### 📌 [1] البيانات الهيكلية والمصنفة من لوحة التحكم:

* **البيانات الرقمية (Numerical Data):** تشمل القيم المالية والكمية القابلة للحساب مثل: سعر المنتج (**250.0 ريال**)، كمية الطلب (**2 قطعة**)، رسوم الشحن (**28.0 ريال**)، وإجمالي الفاتورة الصافية (**528.0 ريال**).
* **البيانات الفئوية (Categorical Data):** تشمل المتغيرات النوعية المحددة مسبقاً لتصنيف الطلب مثل: طريقة الدفع (**تمارا**)، حالة الطلب (**تم الشحن**)، مدينة العميل (**الرياض**)، وفئة المنتج (**عطور**).
* **البيانات النصية الحرة (Text Data):** تشمل النصوص الحرة المرسلة من العميل وتتطلب تحليلاً نصياً متقدماً مثل: اسم العميل كامل، وعنوان التوصيل التفصيلي (حي النرجس)، الملاحظات الخاصة (طلب هدية)، وتقييم المنتج الإيجابي.

> 💡 **ملاحظة المحلل:** تم بنجاح بناء هيكلية القواميس (Dictionaries) في بايثون لفرز وتصنيف البيانات بدقة بناءً على متطلبات اليوم الثاني، مما يمهد الطريق لمعالجة البيانات وفهم سلوك المستهلك بشكل مؤتمت بالكامل.

---

### 🔍 [2] إجمالي الأداء والتحليل الاستراتيجي (Business Insights):

* **تحليل سلوك المتسوق السعودي:** يتبين من البيانات أن العميل اعتمد على حلول الدفع الآجل (**تمارا**)، وهو ما يعكس الاتجاه المتزايد في السوق السعودي للاعتماد على خدمات (اشترِ الآن وادفع لاحقاً - BNPL) لتعزيز القدرة الشرائية وتسهيل المعاملات.
* **القوة الشرائية وسلة المشتريات:** فئة الطلب تتركز في **العطور** بمتوسط سعر مرتفع نسبياً (250 ريال للقطعة)، وبطلب كمية قطعتين في العملية الواحدة، مما يشير إلى جاذبية العروض الحالية أو ارتفاع الولاء للمنتج.
* **الكفاءة اللوجستية وتجربة المستخدم:** تحديث حالة الطلب إلى **"تم الشحن"** إلى العاصمة (الرياض) بالتوازي مع التقييم الممتاز للعميل ("الرائحة ثباتها ممتاز والتوصيل سريع") يدل على كفاءة سلسلة الإمداد وسرعة معالجة الطلبات بالمتجر.

🚀 **التوصية:** بناءً على ملاحظة العميل بأن الطلب عبارة عن "هدية"، نوصي بإضافة خيار "تغليف الهدايا وكروت الإهداء المخصصة" كخدمة إضافية مدفوعة في لوحة تحكم متجر سلة، مما يساهم مباشرة في رفع متوسط قيمة الطلب (AOV) وتلبية تطلعات المستهلكين بشكل أكثر احترافية.

</div>